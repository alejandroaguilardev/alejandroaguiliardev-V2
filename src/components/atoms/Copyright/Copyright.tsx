import { Typography, Grid, Container } from "@mui/material"

export const Copyright = () => {
    return (
        <Grid py={4} sx={{ borderTopWidth: '.1rem', borderTopColor: 'primary.main', borderTopStyle: 'solid' }}>
            <Container maxWidth='lg'>
                <Typography color='primary.contrastText' fontSize={12}>
                    Â© 2020 - 2022 All rights reserved  | Alejandro Aguilar Systems Engineer | Software Developer
                </Typography>
            </Container>
        </Grid>
    )
}
