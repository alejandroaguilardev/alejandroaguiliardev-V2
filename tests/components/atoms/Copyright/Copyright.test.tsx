import { Copyright } from "../../../../src/components/atoms/Copyright/Copyright";
import { render } from '@testing-library/react';

describe("Copyright test", () => {
	test("snapshots", () => {
		const { container } = render(<Copyright />);
		expect(container).toMatchSnapshot();
	});
});
