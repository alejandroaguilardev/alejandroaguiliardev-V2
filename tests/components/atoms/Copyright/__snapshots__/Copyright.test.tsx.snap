// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Copyright test snapshots 1`] = `
<div>
  <div
    class="MuiGrid-root css-815iv4-MuiGrid-root"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
    >
      <p
        class="MuiTypography-root MuiTypography-body1 css-h3yix-MuiTypography-root"
      >
        Â© 2020 - 
        2022
         All rights reserved  | Alejandro Aguilar Systems Engineer | Software Developer
      </p>
    </div>
  </div>
</div>
`;
